src/webviews/mongoClusters/collectionView/components/toolbar/ToolbarMainView.tsx

```diff
@@ -46,6 +66,23 @@ const ToolbarQueryOperations = (): JSX.Element => {
             ...prev,
             currrentQueryDefinition: { ...prev.currrentQueryDefinition },
         }));
+
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'refreshResults',
+                properties: {
+                    ui: 'button',
+                    view: currentContext.currentView,
+                },
+                measurements: {
+                    page: currentContext.currrentQueryDefinition.pageNumber,
+                    pageSize: currentContext.currrentQueryDefinition.pageSize,
+                    queryLength: currentContext.currrentQueryDefinition.queryText.length,
+                },
+            })
+            .catch((error) => {
+                console.error('Failed to report query event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.